### ISSUE TITLE
Running our own instance of Insight Maker

### ISSUE THREAD

**Issue by PierreRustOrange** [scottfr/simulation#3]
Thanks for the great simulation package and Insight Maker tool, I'm using it with some colleagues at Orange, a french ISP ; we really love the tool, it's brillant :)

We would like to use it to run models for which we are not allowed to publish data outside the company. Even if the model is private on Insight Maker, it's unfortunately not enough, therefore we would need to host the tool in our own datacenters.
I've seen that we can run the model on our own servers using this package (i.e. "simulation"), however it lacks the graphic  interface which make it impossible to use for non-developers (I actually am a developer, but most of my colleagues are not).

Long story short, is it possible to host a private instance of insight Make on our own servers ? 

Best Regards and many thanks for making this tool available to all.

**Comment by scottfr** [comment]
Hi! Please use this issue tracker for `simulation` issues only and not questions specific to Insight Maker.

To briefly answer your question though, you can use Insight Maker to build, edit, and run models without saving them to Insight Maker servers. To do that, use the Download/Upload functionality under the Share menu to download a model file to your computer and then load it from your computer. If you just Download/Upload files and don't "Save Insight", your model won't be transmitted to Insight Maker.

You can find that functionality here:

<img width="316" alt="image" src="https://github.com/scottfr/simulation/assets/2190492/dc04de1c-8d85-429b-a56a-da7f06b0b616">

You can also import and run these downloaded model files in `simulation`.

**Comment by PierreRustOrange** [comment]
Many thanks for your reply and sorry for the misplacement of this question.
Is there a place I can ask questions specific to Insight Maker (especially it usage on premises) ?

**Comment by scottfr** [comment]
Please feel free to send questions to me at scott@insightmaker.com
